pylint_protobuf/parse_pb2.py

```diff
@@ -186,22 +190,21 @@ def parse_name(var):
     return parse_name(enum_descriptor)
 
 
+def _get_descriptor_attr(arg):
```

Comment:
This function doesn't appear to be used, it should be removed.

Reply:
Oops, removed!